docs/modules/random.md

```diff
@@ -0,0 +1,400 @@
+# 🎲 Random
+
+**`#bs.random:help`**
+
+Contains many random functions for different distributions and noise patterns.
+
+```{image} /_imgs/modules/random.png
+:align: center
+:class: dark_light p-2
+```
+
+```{epigraph}
+"Creativity is the ability to introduce order into the randomness of nature."
+
+-- Eric Hoffer
+```
+
+---
+
+## 🔧 Functions
+
+You can find below all functions available in this module.
+
+---
+
+### Random choice
+
+```{function} #bs.random:choose
+
+Selects a random value from a list.
+
+:Inputs:
+  **Storage `bs:in random.choose.options`**: {nbt}`list` The list of values to choose from.
+
+:Outputs:
+  **Storage `bs:out random.choose`**: {nbt}`any` The randomly selected value.
+
+  **Return**: The index of the chosen value.
+```
+
+*Pick a random fruit from the list:*
+
+```mcfunction
+# Populate list with values
+data modify storage bs:in random.choose.options set value ["Apples", "Bananas", "Strawberries", "Blueberries", "Mango", "Watermelon", "Honeydew Melon"]
+
+# Randomly select one
+function #bs.random:choose
+
+# Display the result
+tellraw @a [{"text":"Value: ","color":"dark_gray"},{"nbt":"random.choose","storage":"bs:out","color":"gold"}]
+```
+
+> **Credits**: SBtree
+
+
+### Random distributions
```

Comment:
Is it possible to add an image to see the difference between the different distributions?
And I think it could be interesting to add a page/video that explains the distribution for each.